homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+@font-face {
+  font-family: "Actor-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Actor-Regular.woff2") format("woff2"), url("../fonts/Actor-Regular.woff") format("woff");
+}
+@font-face {
+  font-family: "Acme-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Acme-Regular.woff2") format("woff2"), url("../fonts/Acme-Regular.woff") format("woff");
+}
+@font-face {
+  font-family: "Aclonica-Regular";
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/Aclonica-Regular.woff2") format("woff2"), url("../fonts/Aclonica-Regular.woff") format("woff");
+}
+html {
+  box-sizing: border-box;
+}
+*, *::after, *::before {
+  box-sizing: inherit;
+}
+a {
+  text-decoration: none;
+  display: block;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
+}
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+button {
+  padding: 0;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: transparent;
+}
+body {
+  min-width: 375px;
+  overflow-x: hidden;
+  font-family: "Aclonica-Regular", sans-serif;
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+body.lock {
+  overflow: hidden;
+}
+.container {
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: 0 auto;
+  min-width: 375px;
+}
+.hidden {
+  display: none !important;
+}
+.logo__link {
+  font-family: "Actor-Regular";
+  color: #fff;
+  font-size: 26px;
+  line-height: 26px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-weight: bold;
+  letter-spacing: 1.2px;
+}
+.header {
```

Comment:
You have declared header twice, and you never use styles please remove it.

Reply:
done